lib/node_modules/@stdlib/stats/base/dists/triangular/mgf/lib/factory.js

```diff
@@ -75,19 +85,24 @@ function factory( a, b, c ) {
 	* // returns <number>
 	*/
 	function mgf( t ) {
-		var ret;
-
 		if ( isnan( t ) ) {
 			return NaN;
 		}
-		if ( t === 0.0 ) {
-			return 1.0;
+		if ( a < c ) {
+			if ( c < b ) {
+				return (
+					exp( c * t ) * (
+						( ( c - a ) * phi2( ( a - c ) * t ) ) +
+						( ( b - c ) * phi2( ( b - c ) * t ) )
+					) / ( b - a )
+				);
```

Comment:
```suggestion
				return exp( c*t ) * ( ( ( c-a ) * phi2( ( a-c )*t ) ) + ( ( b-c ) * phi2( ( b-c )*t ) ) ) / ( b-a ); // eslint-disable-line max-len
```

Reply:
This is a minor quibble. In short, my preference is to use negative space to visually group related symbols and to draw the eye to the most salient information. This is philosophy underlying our use of negative space throughout the project.